var/spack/repos/builtin/packages/texlive/package.py

```diff
@@ -192,6 +195,10 @@ def build(self, spec, prefix):
 
     @when('@live')
     def install(self, spec, prefix):
+        # The binary install needs a profile file to be present
+        tmp_profile = tempfile.NamedTemporaryFile()
+        tmp_profile.write("selected_scheme {}".format(spec.variants['scheme']).encode())
```

Comment:
```suggestion
        tmp_profile.write("selected_scheme {0}".format(spec.variants['scheme']).encode())
```
Python 2.6 support. May also need to wrap the line for flake8

Reply:
Is there an automatic check for `flake8` compliance?

Reply:
Yes there is. :)